docs/en/wallet-attestation.rst

```diff
@@ -478,7 +478,66 @@ The body of the Wallet Attestation JWT MUST contain:
+| Transition   | Description                                                                                                  |
++==============+==============================================================================================================+
+| `install`    | The Holder performs a fresh installation or restore the initial state of the Wallet Instance on the device   |
++--------------+--------------------------------------------------------------------------------------------------------------+
+| `verify`     | The Wallet Instance has been verified by the Wallet Provider and its Wallet Hardware Key has been registered |
++--------------+--------------------------------------------------------------------------------------------------------------+
+| `validate`   | The Wallet Instance obtains a valid PID                                                                      |
++--------------+--------------------------------------------------------------------------------------------------------------+
+| `invalidate` | The PID expires or gets revoked                                                                              |
++--------------+--------------------------------------------------------------------------------------------------------------+
+| `revoke`     | The Wallet Provider marks the Wallet Instance as not usable                                                  |
++--------------+--------------------------------------------------------------------------------------------------------------+
+| `uninstall`  | The Holder removes the Wallet Instance from the device                                                       |
++--------------+--------------------------------------------------------------------------------------------------------------+
+
+Revocations
+~~~~~~~~~~~~~~~~~~
+As mentioned in the *Wallet Instance initialization and registration* section above, a Wallet Instance is bound to a Wallet Hardware Key and it's uniquely identified by it. 
+The Wallet Provider, which holds the Wallet Hardware Keys, can mark them as *revoked* so to make the related Wallet Instance unusable.
+
+The details of the revocation mechanism used by the Wallet Provider as well as the data model for maintaining the Wallet Instance references is delegated to the Wallet Provider's implementation.
+
+During the *Wallet Instance initialization and registration* phase the Wallet Provider may associate the Wallet Instance with a specific Holder uniquely identified within the Wallet Provider's systems as well as with metadata regarding the device the Wallet Instance is running on such as operative system version, chipset capabilities and the Wallet Solution version. 
+These informations can allow the Wallet Provider to selectively revoke Wallet Instances based on specific criteria.
+
+The choice of which data need to be stored is left to the Wallet Provider.
```

Comment:
Please provide a non-normative example of these data

Reply:
There's no example data to be proposed here as it's delegated to the Wallet Provider's implementation.

That is, each Wallet Provider decides what metadata is collected from the Wallet Instance to provide the safest experience to users along with the possibility to intervene in case of known issues.